docs/TESTS.md

```diff
@@ -0,0 +1,15 @@
+# The `test.tools` framework
```

Comment:
Huh. I actually think we need this one. I checked the site, and we don't add an extra title for this section. Inconsistency :/